Fixed typo in docs/spec/feature(en_US)

nshntarora opened the pull request:
expirerence changed to experience

First of all, thanks for your contribution! :-)

Please makes sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you propose PR to correct branch: bugfix for `master`, feature for latest active branch `feature-x.x`.
* [x] Make sure you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

nshntarora committed:
Fixed typo in docs/spec/feature(en_US)

expirerence changed to experience

yesmeck commented on docs/spec/feature.en-US.md:
```diff
@@ -98,12 +98,8 @@ Beatuful looks, tide typography and smooth animation produce positive reaction a
-
-From 0 to 1, Ant Design help designers to implement a quick prototype for trial and error.
-
-From 1 to 100, Ant Design provide a complete UI solution, help designers to create their own product personality and improve user expirerence.
```
Why these sentences are deleted?

nshntarora replied:
Fixed it. Unintentional. Just a mistake.

nshntarora committed:
Added sentences removed by mistake.

In the previous typo fix commit, a few sentences at the end were removed. This commit fixes that.

yesmeck commented:
Thanks!
